makeRelease.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+
+RELEASE_DIR=${HOME}/Releases
+GITHUB_USER=OpenConext
+PROJECT_NAME=OpenConext-Profile
+
+if [ -z "$1" ]
+then
+
+cat << EOF
+Please specify the tag or branch to make a release of.
+
+Examples:
+
+    sh makeRelease.sh 0.1.0
+    sh makeRelease.sh master
+    sh makeRelease.sh develop
+
+If you want to GPG sign the release, you can specify the "sign" parameter, this will
+invoke the gpg command line tool to sign it.
+
+   sh makeRelease 0.1.0 sign
+
+EOF
+exit 1
+else
+    TAG=$1
+fi
+
+PROJECT_DIR_NAME=${PROJECT_NAME}-${TAG} &&
+PROJECT_DIR=${RELEASE_DIR}/${PROJECT_DIR_NAME} &&
+SYMFONY_ENV=prod
+
+echo "Preparing environment" &&
+mkdir -p ${RELEASE_DIR} &&
+rm -rf ${PROJECT_DIR} &&
+
+echo "Cloning repository";
+cd ${RELEASE_DIR} &&
+git clone https://github.com/${GITHUB_USER}/${PROJECT_NAME}.git ${PROJECT_DIR_NAME} &&
+
+echo "Checking out ${TAG}" &&
+cd ${PROJECT_DIR} &&
+git checkout ${TAG} &&
+
+echo "Running Composer Install";
+SYMFONY_ENV=${SYMFONY_ENV} composer install --no-dev --prefer-dist -o --no-scripts &&
+
+echo "Tagging the release in RELEASE file" &&
+COMMITHASH=`git rev-parse HEAD` &&
+echo "Tag: ${TAG}" > ${PROJECT_DIR}/RELEASE &&
+echo "Commit: ${COMMITHASH}" >> ${PROJECT_DIR}/RELEASE &&
+
+echo "Cleaning build of dev files" &&
+rm -rf ${PROJECT_DIR}/ansible &&
+rm -rf ${PROJECT_DIR}/Vagrantfile &&
+rm -rf ${PROJECT_DIR}/.idea &&
```

Comment:
I copied this from another `makeRelease`. I know there are people that don't have it in their global gitignore, so I left it in, but I agree: they should be gitignored.